packages/wonder-blocks-announcer/types/Announcer.types.ts

```diff
@@ -0,0 +1,37 @@
+/*
+PolitenessLevel: The two options for ARIA Live Regions:
+- polite, which will wait for other announcements to finish
+- assertive, which will interrupt other messages
+*/
+export type PolitenessLevel = "polite" | "assertive";
+
+/*
+RegionFactory: A config for creating duplicate region elements.
+- Count is the total number for each level.
+- aIndex references the index of the last-used assertive log element.
+- pIndex references the index of the last-used polite log element.
+*/
+export type RegionFactory = {
+    count: number;
```

Comment:
**question:** Is count changed at some point? or is it always a static value? I wonder if you can get this number by counting the keys in this factory object instead? (I don't think this is going to change at all, so this approach works too if you find it simpler).